.github/workflows/issue.yml

```diff
@@ -12,8 +12,8 @@ permissions:
 jobs:
   comment:
     permissions:
-      issues: write  # for peter-evans/create-or-update-comment to create or update comment
-      pull-requests: write  # for peter-evans/create-or-update-comment to create or update comment
+      issues: write # for peter-evans/create-or-update-comment to create or update comment
+      pull-requests: write # for peter-evans/create-or-update-comment to create or update comment
```

Comment:
These changes aren't related to the fix. You can undo them by running `git checkout beta -- .github/workflows/issue.yml` and committing the changes to your branch and pushing them.